sema/ro/creator/roc.py

```
@@ -0,0 +1,73 @@
+            indent=indent,
+            auto_compact=auto_compact,
+            **kwargs
+        )
+
+    def _parse_blueprint(self, blueprint_path: str):
+        self.bp = yaml.safe_load(open(blueprint_path, "r")) if blueprint_path else {}
+        version = self.bp.get(":context", {}).get("version", "1.0")
+        self.rocrate_ignore.extend(self.bp.get(":blueprint").get("ignore", []))
+        self.rocrate_include.extend(self.bp.get(":blueprint").get("include", []))
+
+    def _parse_rocrate_ignore(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `_parse_rocrate_ignore`

(ANN202)